server/accounts.go

```diff
@@ -4114,18 +4120,64 @@ type transform struct {
 	src, dest string
 	dtoks     []string
 	stoks     []string
-	dtpi      []int8
+	dtpi      [][]int // destination token position indexes
+	dtpinp    []int32 // destination token position index number of partitions
+}
+
+func getMappingFunctionArgs(functionRegEx *regexp.Regexp, token string) ([]string, error) {
```

Comment:
Yeah, that was a leftover from before Ivan's suggestion to move the regex compilation at initialization time (and therefore with a regex per function name, which means I could then use mustCompile (doesn't return an err) rather than compile (which does)), and therefore now that err is indeed superfluous and can be removed from the signature, will clean up.